shell/pages/c/_cluster/settings/performance.vue

```diff
@@ -56,9 +64,26 @@ export default {
 
       return schema?.resourceMethods?.includes('PUT') ? _EDIT : _VIEW;
     },
+
+    canSave() {
+      return this.value.inactivity.enabled ? this.isInactivityThresholdValid : true;
+    }
   },
 
   methods: {
+    validateInactivityThreshold(value) {
+      if (!this.authUserTTL?.value) {
+        return;
```

Comment:
If there's no authUserTTL validation has basically passed, so need to set `this.isInactivityThresholdValid = false; ` before returning. Without this, currently, when inactivity is enabled the field is always invalid and the changes can never be saved

Reply:
i made a mistake with the suggestion, it should be set to true, updating in incoming commit